src/main/io/gps.c

```diff
@@ -789,6 +797,19 @@ static bool gpsNewFrameNMEA(char c)
                         *gpsPacketLogChar = LOG_NMEA_RMC;
                         gpsSol.groundSpeed = gps_Msg.speed;
                         gpsSol.groundCourse = gps_Msg.ground_course;
+                        // This check will miss 00:00:00.00, but we shouldn't care - next report will be valid
+                        if(!rtcHasTime() && gps_Msg.date != 0 && gps_Msg.time != 0)
```

Comment:
You will have to make your code conditional to `USE_RTC_TIME`, or else it will cause build errors on targets that have GPS but no RTC.